app/models/ingest_job.rb

```diff
@@ -383,23 +394,44 @@ def read_parse_logfile(filepath)
     end
   end
 
-  # generates parse completion email body
+  # generates parse completion email body, and logs analytics to Mixpanel
   #
   # * *returns*
   #   - (Array) => List of message strings to print in a completion email
-  def generate_success_email_array
+  def get_email_and_log_to_mixpanel
+    file_type = self.study_file.file_type
+
     message = ["Total parse time: #{self.get_total_runtime}"]
-    case self.study_file.file_type
+
+    # Event properties to log to Mixpanel.  Mixpanel uses camelCase for props.
+    mixpanel_log_props = {
```

Comment:
Just a style comment - I know this is getting logged externally to Mixpanel, but the Ruby convention for variable names is `snake_case`, as opposed to `camelCase`.  Not a blocking concern.

Reply:
Yup, by "Mixpanel uses camelCase for props", I mean using Ruby-conventional casing here would cause inconsistent UX in Mixpanel.  (The Mixpanel analyst would need to switch between cases, depending on the language of the logging agent.)  I've refined the comment in 6265f13.

Reply:
Ah, understood.  I missed that comment.